kfs-ld/src/main/java/edu/arizona/kfs/module/ld/LaborConstants.java

```diff
@@ -25,4 +25,7 @@
 	
 	public static final String[] BALANCE_TYPES = {"AC", "CB", "IE"};
 
+	//SalaryExpenseTransferDocument
+	public static final String BALANCE_TYPE_ACTUAL = "AC";
+	public static final String BALANCE_TYPE_A2 = "A2";
```

Comment:
Please remove unneeded imports and properties for OriginEntryFull, OptionsService, UniversityDateService, OptionsServiceImpl, UniversityDateServiceImpl from SalaryExpenseTransferDocument.java.